### PR TITLE
fixed all for flake8

### PR DESCRIPTION
**Pull request by lericdev**

### CODE REVIEW

File: kpi_notificator/settings/prod.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-from .base import *
```
Comment (bloogrox) [inline kpi_notificator/settings/prod.py:1]: add at the end `# noqa`
```
from .base import *  # noqa
```

File: manage.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
         # issue is really that Django is missing to avoid masking other
         # exceptions on Python 2.
         try:
-            import django
+            # import django
```
Comment (bloogrox) [inline manage.py]: you cannot remote django import

add `# noqa` at the end

```
import django  # noqa
````

File: stats/__init__.py [hunk 3]
```diff
@@ -1 +1 @@
-from .receivers import on_trigger
\ No newline at end of file
+# from .receivers import on_trigger
```
Comment (bloogrox) [inline stats/__init__.py:1]: wrong
add `# noqa` at the end

**Review by bloogrox — CHANGES_REQUESTED** [review]

File: stats/__init__.py (hunk 3 above)
Comment (lericdev) [reply stats/__init__.py:1]: Ok, thank you.

**Pull request closed without merging**